python/vowpalwabbit/sklearn_vw.py

```diff
@@ -347,7 +349,14 @@ def predict(self, X):
             Output vector relative to X.
         """
 
-        check_is_fitted(self, 'fit_')
+        if sklearn.__version__ < '0.22':
```

Comment:
In sklearn <0.22, the second parameter is required. In sklearn ==0.22, it does nothing, and in sklearn >=0.23 it will be removed altogether. We need to do a version check to maintain compatibility between all three versions.

The error message being thrown is copied from the sklearn check_is_fitted function